examples/angular/src/pages/ui/components/authenticator/custom-slots/custom-slots.component.ts

```diff
@@ -15,45 +15,38 @@ export class CustomSlotsComponent {
   public formFields = {
     signIn: {
       username: {
-        labelHidden: false,
         placeholder: 'Enter your email',
       },
     },
     signUp: {
       password: {
-        labelHidden: false,
         label: 'Password:',
         placeholder: 'Enter your Password:',
```

Comment:
Question, and this might be more of a question to @dbanksdesign, but should we hide placeholder if we're showing the label?

Reply:
FWIW don't think we should hide placeholders, they are useful for instructional messages

Reply:
(I realize I posted this at the wrong code snippet, I'll continue the comments in other thread).